proxy/src/main/java/com/velocitypowered/proxy/command/SuggestionsProvider.java

```diff
@@ -357,7 +358,10 @@ private static boolean shouldConsider(final String name, final String input) {
     return CompletableFuture.allOf(futures).handle((unused, throwable) -> {
       final List<Suggestions> suggestions = new ArrayList<>(futures.length);
       for (final CompletableFuture<Suggestions> future : futures) {
-        if (!future.isCompletedExceptionally()) {
+        if (future.isCompletedExceptionally()) {
+          final Throwable exception = CompletableFutures.getException(future);
```

Comment:
By the way, sometimes this exception, which is the cause of the CompletionException, can be null, but it's quite bad practice, requiring a subclass of CompletionException which adds no cause. I don't think such case is worth guarding against.

Reply:
As far as I can see, log4j will [handle the null `Throwable`](https://github.com/apache/logging-log4j2/blob/e395e13c513e17cb91781808f83b20ee559bf534/log4j-api/src/main/java/org/apache/logging/log4j/simple/SimpleLogger.java#L240) just fine.